lib/travis/build/script/go.rb

```diff
@@ -103,6 +108,13 @@ def uses_make
             '-f GNUmakefile || -f makefile || -f Makefile || -f BSDmakefile'
           end
 
+	  def uses_15_vendoring
+            if (go_version < '1.5' || go_version == 'go1')
+              'false'
+            end
+	    (go_version < '1.6') ? '$GO15VENDOREXPERIMENT == 1' : '$GO15VENDOREXPERIMENT != 0'
```

Comment:
IMO (not maintainer, just an outside observation) wouldn't it make sense to include a link to https://golang.org/doc/go1.6#go_command here? (so it's more clear why this is ternary based on whether we're `> '1.6'`)

Also, will this `go_version` string comparison work if/when Go reaches 1.10?  Is there some stdlib Ruby version library that could be used instead?